Reqnroll/ErrorHandling/BindingException.cs

```diff
@@ -7,7 +7,7 @@ namespace Reqnroll
     [Serializable]
```

Comment:
Some [Serializable] attributes has been removed but this one is still here. What is the plan with this attribute?

Reply:
https://github.com/reqnroll/Reqnroll/issues/736